From 9ba1c22be8ecb2617b11c248aa52ea79ec1ffec0 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 6 Jul 2018 12:29:43 +0200
Subject: [PATCH 23/24] Merge tag 'android-8.1.0_r36' into
 staging/lineage-15.1-android-8.1.0_r36

Android 8.1.0 Release 36 (OPM2.171026.006.H1)

* tag 'android-8.1.0_r36':
  DO NOT MERGE: Update Platform Security String to 2018-07-05 for oc-mr1-dev Bug:79883349 (cherry picked from commit cfbe84b8a0ac6ad680edad1c21b08672ed96127d)
  DO NOT MERGE: Updating security string for oc-mr1-dev to 2018-06-05

Change-Id: If99e4adeba19295ec9b8d33e21b4e4a841b63c3a
---
 core/build_id.mk         | 2 +-
 core/version_defaults.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index da0b64364..973767d48 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=OPM2.171026.006.C1
+export BUILD_ID=OPM2.171026.006.H1
diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index efe68764f..e23b9af30 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -181,7 +181,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2018-06-05
+      PLATFORM_SECURITY_PATCH := 2018-07-05
 endif
 
 ifndef PLATFORM_BASE_OS
-- 
2.17.1

